future-apps/payroll/contracts/Payroll.sol

```diff
@@ -348,7 +348,7 @@ contract Payroll is EtherTokenConstant, IForwarder, IsContract, AragonApp {
         employeeIds[_newAddress] = employeeId;
         employee.accountAddress = _newAddress;
 
-        emit ChangeAddressByEmployee(employeeId, oldAddress, _newAddress);
+        emit ChangeAddressByEmployee(employeeId, _newAddress, oldAddress);
```

Comment:
I've flipped the ordering to be more similar to the order for roles (new value, old value)